libraries/adaptive-expressions-ie11/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from 'adaptive-expressions';
```

Comment:
I feel a bit silly for saying this, but can you add the appropriate copyright headers to the file?

```suggestion
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

export * from 'adaptive-expressions';
```